From d457ac80b10c2f5c809fce61cbc1591ec08ca472 Mon Sep 17 00:00:00 2001
From: Song Hui <hui.song_1@nxp.com>
Date: Thu, 8 Aug 2019 18:10:41 +0800
Subject: [PATCH 568/741] gpio: mpc8xxx: add ls1088a platform special function

commit 32bc0c1ee5ce91c0390603154fdf7b7bceee5e78 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

ls1028a and ls1088a platform share common special function.
The gpio hardware what they use is the same version.

Signed-off-by: Song Hui <hui.song_1@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/gpio/gpio-mpc8xxx.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpio/gpio-mpc8xxx.c b/drivers/gpio/gpio-mpc8xxx.c
index de00d1f2182d..b332f384d038 100644
--- a/drivers/gpio/gpio-mpc8xxx.c
+++ b/drivers/gpio/gpio-mpc8xxx.c
@@ -319,6 +319,7 @@ static const struct of_device_id mpc8xxx_gpio_ids[] = {
 	{ .compatible = "fsl,mpc5125-gpio", .data = &mpc5125_gpio_devtype, },
 	{ .compatible = "fsl,pq3-gpio",     },
 	{ .compatible = "fsl,ls1028a-gpio", .data = &ls1028a_gpio_devtype, },
+	{ .compatible = "fsl,ls1088a-gpio", .data = &ls1028a_gpio_devtype, },
 	{ .compatible = "fsl,qoriq-gpio",   },
 	{}
 };
-- 
2.17.1

